com.Acestream.engine.json

```diff
@@ -0,0 +1,149 @@
+{
+    "app-id": "com.Acestream.engine",
+    "runtime": "org.gnome.Platform",
```

Comment:
Why do you use gnome runtime instead of freedesktop? I don't see what gnome dependency this app has.